src/kedro_neptune/__init__.py

```diff
@@ -434,11 +434,14 @@ def __init__(self):
 
     @hook_impl
     def after_catalog_created(self, catalog: DataCatalog) -> None:
-        self._run_id = hashlib.md5(str(time.time()).encode()).hexdigest()
+        self._run_id = (
+            os.environ.get("NEPTUNE_CUSTOM_RUN_ID")
+            or hashlib.md5(str(time.time()).encode()).hexdigest()
+        )
 
         config = get_neptune_config(settings)
 
-        if config.enabled:
+        if config.enabled and not os.environ.get("NEPTUNE_CUSTOM_RUN_ID"):
```

Comment:
As far as I know `os.getenv` is more pythonic than `os.environ.get` - I think we should use it here

Reply:
And in the modified line above